Sources/Sentry/SentryOptions.m

```diff
@@ -23,8 +23,7 @@ @implementation SentryOptions
         @"SentryCrashIntegration", @"SentryFramesTrackingIntegration",
         @"SentryAutoBreadcrumbTrackingIntegration", @"SentryAutoSessionTrackingIntegration",
         @"SentryAppStartTrackingIntegration", @"SentryOutOfMemoryTrackingIntegration",
-        @"SentryPerformanceTrackingIntegration", @"SentryNetworkTrackingIntegration",
-        @"SentryFileIOTrackingIntegration"
```

Comment:
h: Probably a merge problem, SentryFileIOTrackingIntegration should not be removed. That's why all File IO Tracking tests are falling.

Reply:
Great catch, this makes sense.